docs/shell-tips.md

```diff
@@ -49,6 +49,36 @@ $ cd ~
 $ cd /home/john
 ```
 
+## Change directory (`cd`)
+
+When you're using the shell, you work inside some directory (the one
+specified in the prompt). This way you can point to files relative to your
+current directory, instead of writing the whole path.
+
+Imagine you have a file called `ideas.txt` inside `/home/john/notes/`, and
+you want to edit it using `nano`. You could use:
+
+```
+$ nano /home/john/notes/ideas.txt
+```
+
+However, that isn't very practical, especially if you are working with
+longer paths.
+
+That's why it's very useful to change the path where you are currently
+located (usually known as **working directory**). To do that, you use `cd`
+(**c**hange **d**irectory):
+
+```
+$ cd /home/john/notes/ideas.txt
+~/notes$ nano ideas.txt
+```
+
+Or, if you're the user `john`: `cd ~/notes`.
+
+Thanks to this, you can access to all the files inside `/home/john/notes`
+directly, without needing to type the whole path.
+
```

Comment:
What about pointing to [this](http://www.linuxnix.com/abslute-path-vs-relative-path-in-linuxunix/) article on relative/absolute paths? I think the examples are very clear and easy to understand.